src/java/main/org/apache/zookeeper/ZooDefs.java

```diff
@@ -96,21 +97,20 @@
         /**
          * This is a completely open ACL .
          */
-        public final ArrayList<ACL> OPEN_ACL_UNSAFE = new ArrayList<ACL>(
-                Collections.singletonList(new ACL(Perms.ALL, ANYONE_ID_UNSAFE)));
+        public final List<ACL> OPEN_ACL_UNSAFE =
+                Collections.singletonList(new ACL(Perms.ALL, ANYONE_ID_UNSAFE));
 
         /**
          * This ACL gives the creators authentication id's all permissions.
          */
-        public final ArrayList<ACL> CREATOR_ALL_ACL = new ArrayList<ACL>(
-                Collections.singletonList(new ACL(Perms.ALL, AUTH_IDS)));
+        public final List<ACL> CREATOR_ALL_ACL =
```

Comment:
Same as above, typically we won't modify the exposed APIs in 3.4.* releases. Its OK to exclude this case as well.